tasks/sign-base64-blob/README.md

```diff
@@ -25,3 +25,7 @@ data:
         pipelineImage: <image pullspec>
         configMapName: <configmap name>
 ```
+## Changelog
```

Comment:
we don't typically add this line

Reply:
Fixed! I probably copied it from an old task